frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/profile-settings-button.tsx

```diff
@@ -0,0 +1,44 @@
+import { Icon } from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/icon-color.enum.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback } from '#libs/hooks/hooks.js';
+import { type IconName } from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: React.ReactNode;
+  name: IconName;
+  onClick: (name: IconName) => void;
+  isActive?: boolean;
+};
+
+const ProfileSettingsButton: React.FC<Properties> = ({
```

Comment:
Can't we reuse Card component for this?

Reply:
There was not card component initially but I will see how to utilise it right now

Reply:
@Liza-Veis 
I went over the Card component trying to figure out how to implement it instead of the ProfileSettingsButton component. Right now, I can't say that I could just apply it and roll with it. 
Card component uses img and does not utilise the Icon component we developed, it does not account for an Icon to have a background either, it also does not use the arrow icon for marking active button, it does not account for the additional icon at all (Primary icon of the button + arrow icon or trash bin icon) 
But I do like the idea of having this card universal for all components. If you tell me to, I can work on the Card component to make it account for all use cases. I'm just worried about the fact that it may significantly postpone the merge of my PR.

Reply:
let's add iconName property and if it's passed while image not, render icon instead